src/components/ProfileFormUpdate/index.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import gql from 'graphql-tag';
+import { Mutation, Query } from 'react-apollo';
+import { Formik } from 'formik';
+import { Form, Message } from 'semantic-ui-react';
+import { CenterWrapper, SpinnerWrapper } from '../../styles/App';
+import Spinner from '../Spinner';
+import TextInput from '../TextInput';
+import AsyncButton from '../AsyncButton';
+
+const ProfileFormUpdate = () => (
+  <Query query={GET_PROFILE_QUERY}>
+    {({ data, loading, error }) => {
+
+      if (loading) {
+        return (
+          <SpinnerWrapper bgColor="#FFFFFF">
+            <Spinner />
+          </SpinnerWrapper>
+        )
+      }
+
+      if (error) return <div>Error: {error.message}</div>;
+
+      const { id, avatar, email, address, phone } = data.me;
+
+      return (
+        <Mutation
+          mutation={EDIT_PROFILE_MUTATION}
+          // update={(store, { data: { updateDailyReport } }) => {
```

Comment:
I think it's better to delete line 30 ~ 38 or to write `update` for profile

Reply:
Yeah, a small my mistake. I'll delete that. Thank for remind !

Reply:
Sorry for annoying, but thanks m(_ _)m